src/polycap-description.c

```diff
@@ -84,6 +85,17 @@ polycap_description* polycap_description_new_from_file(const char *filename)
 		exit(1);
 	}
 
+	*source = malloc(sizeof(polycap_source));
```

Comment:
Hey Pieter, 

Blij te zien dat je weer aan polycap kan werken 😄 

Een memory leak: uw malloc voor `*source` is nutteloos aangezien je toch gaat werken met `source_temp`.

Reply:
Ja, was even wat drukker met FWO aanvra(a)g(en) en wat IDL code om WD spectra te fitten, maar nu heb ik dus weer wat meer tijd :)

Good catch. Will fix soon.